tests/UtilitiesTests.cpp

```diff
@@ -0,0 +1,65 @@
+#define _USE_MATH_DEFINES
+
+#include "Fixtures.h"
+#include "UsgsAstroPlugin.h"
+#include "Utilities.h"
+
+#include <json.hpp>
+#include <gtest/gtest.h>
+
+#include <math.h>
+
+using json = nlohmann::json;
+
+
+TEST(UtilitiesTests, calculateRotationMatrixFromEuler) {
+   double euler[3], rotationMatrix[9];
+   euler[0] = 0;
+   euler[1] = M_PI/2;
+   euler[2] = 0;
+   calculateRotationMatrixFromEuler(euler, rotationMatrix);
+   EXPECT_NEAR(rotationMatrix[0], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[1], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[2], 1, 1e-8);
+   EXPECT_NEAR(rotationMatrix[3], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[4], 1, 1e-8);
+   EXPECT_NEAR(rotationMatrix[5], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[6], -1, 1e-8);
+   EXPECT_NEAR(rotationMatrix[7], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[8], 0, 1e-8);
```

Comment:
These should be EXPECT_DOUBLE_EQ and not EXPECT_NEAR

Reply:
EXPECT_NEARs are needed for precision issue here. Added comments to explain as much.